Src/Plugins/Tests.Security/TestData/ReviewPotentiallySensitiveData/ExpectedOutputs/SEC102_001.EmailAddress_with_email_addresses.sarif

```diff
@@ -283,47 +283,6 @@
             "ValidationFingerprint/v2": "{\"host\":\"example.com\",\"id\":\"+lastname\"}"
           }
         },
-        {
-          "ruleId": "SEC102/001",
-          "ruleIndex": 0,
-          "level": "note",
-          "message": {
-            "id": "Default",
-            "arguments": [
-              "lastname",
-              "example.com",
-              "email address"
-            ]
-          },
-          "locations": [
-            {
-              "physicalLocation": {
-                "artifactLocation": {
-                  "uri": "src/Plugins/Tests.Security/TestData/ReviewPotentiallySensitiveData/Inputs/SEC102_001.EmailAddress_with_email_addresses.txt",
-                  "uriBaseId": "SRC_ROOT"
-                },
-                "region": {
-                  "startLine": 4,
-                  "startColumn": 11,
-                  "endLine": 4,
-                  "endColumn": 31,
-                  "charOffset": 277,
-                  "charLength": 20,
-                  "snippet": {
-                    "text": "lastname@example.com"
-                  }
-                }
-              }
-            }
-          ],
-          "fingerprints": {
-            "AssetFingerprint/v1": "[host=example.com][id=lastname]",
-            "ValidationFingerprint/v1": "[host=example.com][id=lastname]",
-            "ValidationFingerprintHash/v1": "a82a08b8f8cd5a7eabf30386974158e85fa1f0982b13d273aa94373c2638eb42",
-            "AssetFingerprint/v2": "{\"host\":\"example.com\",\"id\":\"lastname\"}",
-            "ValidationFingerprint/v2": "{\"host\":\"example.com\",\"id\":\"lastname\"}"
-          }
```

Comment:
>} [](http://example.com/codeflow?start=10&length=1)

Why has this stuff gone away?

Reply:
we weren't using ValidatorBase, so duplicated results would appear.
With my current change, we will ask for the static or dynamic validator class, removing duplications